<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1478097551816" stop="1478097561828">
	<name>a</name>
	<title>a</title>
	<test-cases>
		<test-case start="1478097551818" stop="1478097561826" status="broken">
			<name>sasd</name>
			<title>sasd</title>
			<labels>
			</labels>
			<parameters>
				<parameter kind="environment-variable" name="capabilities" value="{&quot;browserName&quot;:&quot;phantomjs&quot;}"/>
				<parameter kind="environment-variable" name="spec files" value="[&quot;/Users/christianbromann/Sites/Webprojekte/webdriverjs/DEV/a.js&quot;]"/>
			</parameters>
			<steps>
				<step start="1478097551827" stop="1478097552211" status="passed">
					<name>POST /wd/hub/session/42f039b4-99f4-46ac-970f-0632a0f91519/url</name>
					<title>POST /wd/hub/session/42f039b4-99f4-46ac-970f-0632a0f91519/url</title>
					<attachments>
						<attachment title="Request" source="e66e65b7-b74a-44ef-b408-5a070f8b4fbf-attachment.json" type="application/json" size="57"/>
						<attachment title="Response" source="6b98f6ba-c037-4c35-8e68-6d92e0874178-attachment.json" type="application/json" size="195"/>
					</attachments>
					<steps>
					</steps>
				</step>
				<step start="1478097552257" stop="1478097552307" status="passed">
					<name>POST /wd/hub/session/42f039b4-99f4-46ac-970f-0632a0f91519/elements</name>
					<title>POST /wd/hub/session/42f039b4-99f4-46ac-970f-0632a0f91519/elements</title>
					<attachments>
						<attachment title="Request" source="cb588d28-d1f8-44f2-89dd-d576b297b16c-attachment.json" type="application/json" size="51"/>
						<attachment title="Response" source="0461f2be-1b52-4fda-b790-54b4d12fa58a-attachment.json" type="application/json" size="1657"/>
					</attachments>
					<steps>
					</steps>
				</step>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.</message>
				<stack-trace>Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
    at Timeout.&lt;anonymous&gt; (/Users/christianbromann/Sites/Webprojekte/webdriverjs/wdio-mocha-framework/node_modules/mocha/lib/runnable.js:232:19)
    at tryOnTimeout (timers.js:224:11)
    at Timer.listOnTimeout (timers.js:198:5)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>